.github/workflows/deployTest.yml

```diff
@@ -14,105 +14,105 @@ env:
-      ARM_CLIENT_SECRET: ${{ secrets.TERRAFORM_ARM_CLIENT_SECRET }}
-      ARM_SUBSCRIPTION_ID: ${{ secrets.TERRAFORM_ARM_SUBSCRIPTION_ID }}
-      ARM_TENANT_ID: ${{ secrets.TERRAFORM_ARM_TENANT_ID }}
-      OKTA_API_TOKEN: ${{ secrets.OKTA_API_TOKEN_NONPROD }}
-    steps:
-      - uses: actions/checkout@v3
-      - uses: azure/login@v1
-        with:
-          creds: ${{ secrets.AZURE_CREDENTIALS }}
-      - uses: hashicorp/setup-terraform@v2.0.3
-        with:
-          terraform_version: 1.3.3
-      - name: Build ReportStream function app
-        uses: ./.github/actions/build-reportstream-functions
-        with:
-          deploy-env: ${{env.DEPLOY_ENV}}
-      - name: Terraform Init
-        run: make init-${{ env.DEPLOY_ENV }}
-      - name: Terraform deploy (infrastructure and staging slot)
-        run: make deploy-${{ env.DEPLOY_ENV }}
-      - name: Wait for correct commit to be deployed in staging slot
-        timeout-minutes: 5
-        run: make wait-for-${{ env.DEPLOY_ENV }}-slot-commit
-      - name: Wait for staging deploy to be ready
-        timeout-minutes: 1
-        run: make wait-for-${{ env.DEPLOY_ENV }}-slot-readiness
-  build-frontend:
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v3
-      - uses: actions/setup-node@v3.6.0
-        with:
-          node-version: ${{env.NODE_VERSION}}
-      - name: Use cache for node_modules
-        uses: actions/cache@v3.2.5
-        with:
-          path: |
-            ./frontend/node_modules
-          key: npm-${{env.NODE_VERSION}}-${{ hashFiles('frontend/yarn.lock', 'frontend/package.json') }}
-      - uses: azure/login@v1
-        with:
-          creds: ${{ secrets.AZURE_CREDENTIALS }}
-      - uses: ./.github/actions/build-frontend
-        name: Build front-end application
-        with:
-          deploy-env: ${{env.DEPLOY_ENV}}
-          smarty-streets-key: ${{ secrets.SMARTY_STREETS_KEY }}
-          client-tarball: ./client.tgz
-          okta-enabled: true
-          okta-url: https://hhs-prime.oktapreview.com
-          okta-client-id: 0oa1khettjHnj3EPT1d7
-      - name: Save compiled frontend application
-        uses: actions/upload-artifact@v3
-        if: success()
-        with:
-          name: frontend-tarball
-          path: client.tgz
-          retention-days: 1
-  deploy:
-    runs-on: ubuntu-latest
-    environment:
-      name: Test
-      url: https://test.simplereport.gov
-    needs: [build-frontend, prerelease-backend]
-    steps:
-      - uses: actions/checkout@v3
-      - uses: azure/login@v1
-        with:
-          creds: ${{ secrets.AZURE_CREDENTIALS }}
-      - name: Retrieve frontend build
-        uses: actions/download-artifact@v3
-        with:
-          name: frontend-tarball
-      - name: Promote and deploy
-        uses: ./.github/actions/deploy-application
-        with:
-          client-tarball: client.tgz
-          deploy-env: ${{env.DEPLOY_ENV}}
+  # build-docker:
```

Comment:
I am assuming this is commented out to save time since the deploy step doesn't work, does it make sense to just disable the job via adding a 
`if: ${{ false }}` condition rather than commenting

Reply:
Both accomplish the goal of disabling these jobs.

Reply:
I agree, they are indeed functionally the same. But commenting out code is generally considered not a best practice because of two reasons:

- the comment feature should be used to comment/document code, committing commented out code is not a best practice
- most people consider commented out code as dead code, and thus should be removed when doing a code clean up. In this case, I don't believe the code is dead, we just need to disable the job

Reply:
My two cents as a lead:

Stylistic concerns are noted. In this case, the function of the commented code is clearly delineated, with a ticket already created to track the other item. This is strictly temporary, and will be resolved in a matter of days. 

We can argue style all day and get nowhere. This solution works for what we need it to do in the timeframe it needs to be active.

Respectfully, let's move on.

Reply:
I understand that some have strong opinions about commented-out code and its purpose. I may be a little more flexible with it.

My arguments for these comments not being an issue to care about in this case are the following.

- This is temporary.
- This is easier, and the amount of time and overhead to add a conditional to every step in this file is far more than simply commenting out the jobs.
- With the if statement you suggest, our pipelines have to provision Github runners and run the checks to figure out if they have jobs to run, which seems like a waste to me.

I've gone ahead and deleted the code.

Reply:
Thanks for making the change @alismx